geode-core/build.gradle

```diff
@@ -354,6 +355,7 @@ dependencies {
   integrationTestImplementation(project(':geode-dunit'))
   integrationTestImplementation(project(':geode-log4j'))
   integrationTestImplementation(project(':geode-concurrency-test'))
+  integrationTestImplementation(project(':geode-serialization'))
```

Comment:
should not be needed, since `integrationTestImplementation` extends from `implementation` but I like the precision of adding it.